x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/question_answering/question_answering_inference.ts

```diff
@@ -54,9 +57,19 @@ export class QuestionAnsweringInference extends InferenceBase<QuestionAnsweringR
         'Provide a question and test how well the model extracts an answer from your input text.',
     }),
   ];
-
   public questionText$ = new BehaviorSubject<string>('');
 
+  constructor(
+    trainedModelsApi: ReturnType<typeof trainedModelsApiProvider>,
+    model: estypes.MlTrainedModelConfig,
+    inputType: INPUT_TYPE
+  ) {
+    super(trainedModelsApi, model, inputType);
+
+    this.validators$.push(this.questionText$.pipe(map((questionText) => questionText !== '')));
+    this.initializeValidators();
```

Comment:
maybe the `initializeValidators` method should accept a collection of validators? in that case, there is no need to explicitly access `this.validators$`.  
```suggestion
    this.initializeValidators([this.questionText$.pipe(map((questionText) => questionText !== ''))]);
```

Reply:
yes I like this idea.
Added in https://github.com/elastic/kibana/pull/145357/commits/dc96bf33cb9f99bec1b3a85fca04b704ce006186